sdks/unity/AgonesSdk.cs

```diff
@@ -212,6 +223,26 @@ public async Task<bool> Reserve(TimeSpan duration)
             return await SendRequestAsync("/reserve", json).ContinueWith(task => task.Result.ok);
         }
 
+        /// <summary>
+        /// Start sending health ping to the Agones sidecar. 
+        /// </summary>
+        public void StartHealthChecks()

```

Comment:
Thanks for sending this through. I'm curious though - what's the use case for these functions?

Was there a particular scenario in which you would want to manually start the health check process, rather than have it be part of the component?

Also, when would you want to stop a health check?

Reply:
> when would you want to stop a health check?

In my particular case, I do resue my game servers for a long period of time and they all have two external connections with Backend Servers (economy, rewards, etc..) and Party Servers (matchmaking, tickets, backfill).
In case if the GS loses the connection I mark it as non-healthy until the connection is restored.

There may be other use cases like the first match is complete but GS can't load the resources to continue.

> Was there a particular scenario in which you would want to manually start the health check process, rather than have it be part of the component?

Yes, like GS was able to restore the connection or load the resources or some background process was revived.

Reply:
> In case if the GS loses the connection I mark it as non-healthy until the connection is restored.

This sounds like a very dangerous approach -- I wouldn't advise it. Unhealthy game servers have failed a health check and are deemed to not be alive. If this GameServer was part of a Fleet, it would immediately be shut down and recreated.

> Yes, like GS was able to restore the connection or load the resources or some background process was revived.

I can't advocate for any of these scenarios.

This could be solved using custom labels for example, without abusing health checking abilities.

Reply:
Hmm, you are right, this PR can be removed.